consensus/clique/src/main/java/org/hyperledger/besu/consensus/clique/blockcreation/CliqueMiningCoordinator.java

```diff
@@ -33,6 +41,30 @@ public CliqueMiningCoordinator(
     this.miningTracker = miningTracker;
   }
 
+  @Override
+  protected void inSyncChanged(final boolean inSync) {
+    synchronized (this) {
+      final VoteTally validatorProvider =
+          miningTracker
+              .getProtocolContext()
+              .getConsensusState(CliqueContext.class)
+              .getVoteTallyCache()
+              .getVoteTallyAfterBlock(blockchain.getChainHeadHeader());
+      final boolean isValidator =
+          getCoinbase()
+              .filter(coinbase -> validatorProvider.getValidators().contains(coinbase))
+              .isPresent();
+
+      if (inSync && startMiningIfPossible() && isValidator) {
```

Comment:
nit: maybe isSigner? IBFT2 has validators, Clique has signers.

Reply:
I created an `isSigner` method in CliqueHelpers